packages/types/src/primitive/index.ts

```diff
@@ -14,6 +14,9 @@ export { default as AccountId, AccountIdOf } from './AccountId';
 export { default as AccountIndex } from './AccountIndex';
 export { default as AccountInfo } from './AccountInfo';
 export { default as Address } from './Address';
+// Balance is technically not a primitive, however since it is used in a number of interfaces
```

Comment:
This is where the `ClassOf` definition above can come in handy - but that is a phase II. Only to be done once all simple types are moved to "non-constructable" and the primitives can be trimmed to be "only real primitives, some of these in here are questionable as well)